nw-firestore-schema.ts

```diff
@@ -167,6 +167,7 @@ interface Livesite {
   judgingOpen: boolean; // particpants can judge other projects
   judgingReleased: boolean; // participants can see their judging results
   submissionsOpen: boolean; // participants can submit projects
+  applicationsOpen: boolean; // hackers can start applying
```

Comment:
While we're at it, should we add an `applicationsClose` field too?

Reply:
isn't this just a boolean flag? so `applicationClosed` is just `applicationsOpen: false`

Reply:
wait my bad I'm not thinking straight. I keep on thinking it's a timestamp :mildpanic: ok nvm this is good.